Solutions/Microsoft 365 Defender/Hunting Queries/Malicious Emails delivered.yaml

```diff
@@ -0,0 +1,13 @@
+id: cdac93ef-56c0-45bf-9e7f-9cbf0ad06456
+name: Malicious Emails delivered due to User/Admin Overrides
```

Comment:
I think this query can be combined with EmailDelivered-ToInbox case itself with a bool value for Org or User level action.

Reply:
Completed

Reply:
Malicious emails delivered hunting query removed